charts/jenkins/unittests/jcasc-config-test.yaml

```diff
@@ -65,7 +65,7 @@ tests:
                           - envVar:
                               key: "JENKINS_URL"
                               value: "http://RELEASE-NAME-jenkins.default.svc.cluster.local:8080/"
-                        image: "jenkins/inbound-agent:4.11.2-4"
+                        image: "jenkins/inbound-agent:3107.v665000b_51092-5"
```

Comment:
this image doesn't exist:

```
docker pull jenkins/inbound-agent:3107.v665000b_51092-5
Error response from daemon: manifest for jenkins/inbound-agent:3107.v665000b_51092-5 not found: manifest unknown: manifest unknown
```

I keep getting emails from artifacthub about it failing to scan it

Reply:
Right, looks like something changed within the deployment of these images. There's no image similar to `4.11.2-4` left in https://hub.docker.com/r/jenkins/inbound-agent/tags?page=1&name=3107.v665000b_51092-5 👀

Reply:
at a guess `-5` failed on linux publishing, but windows passed. I can see `-4`